src/libraries/Common/tests/System/IO/Compression/CompressionStreamUnitTestBase.cs

```diff
@@ -662,7 +662,7 @@ public static IEnumerable<object[]> RoundtripCompressDecompressOuterData
         }
 
         [Fact]
-        public async Task CompressDecompress_RoundTrip()
+        public async Task CompressDecompress_RoundTrip_BaseTests()
```

Comment:
I'd prefer to see this test's name left as-is, and the method it delegates to instead be renamed to something like CompressDecompress_RoundTrip_OuterLoop.  Same goes for the other similar cases here.  My reason is "BaseTests" doesn't really speak to what the test is doing, plus this is the test that runs 99% of the time, and the one it delegates to is relegated to only running when we run "OuterLoop".